styles.css

```diff
@@ -399,3 +401,181 @@ span {
 footer {
   padding-top: 20px;
 }
+
+#navbar-grid {
+  display: none;
+}
+
+#desktop-contactform {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  #navbar-grid {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+  }
+
+  #desktop-navbar {
+    grid-column: 6 / span 6;
+    justify-content: center;
+  }
+
+  #desktop-contactform {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background: url(assets/contactbg.PNG);
+    background-repeat: no-repeat;
+    background-size: 100% 90%;
+    width: 100%;
+    padding-bottom: 100px;
+    margin-top: 60px;
+  }
+
+  #desktop-navbar a {
+    color: #3a4a42;
+    margin-left: 15px;
+    font-weight: bold;
+  }
+
+  #projects-card {
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 30px;
+    padding-left: 50px;
+    padding-right: 50px;
+  }
```

Comment:
- [ ] Good job for adjust the padding 👍👍 While fixing the padding, I noticed that the column gap is more than it was present in the previous review. Kindly fix this 🤓⛏ You can do this by changing the `column-gap` to `20px`

Your design before | Your design now
---| ---|
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/106119105/7e1b371d-acc2-4e0e-930d-4c3f076a2b0d) | ![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/106119105/3403124e-be95-42e1-8d0a-ba8d357882be)